tests/test_cli_secrets.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2019 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA client secrets tests."""
+
+from click.testing import CliRunner
+from mock import Mock, patch
+from pytest_reana.test_utils import make_mock_api_client
+
+from reana_client.cli import cli
+
+
+def test_secrets_list_server_not_reachable():
```

Comment:
I have created an issue, we should test this only once when we have centralised the check, to do later https://github.com/reanahub/reana-client/issues/287.